### ISSUE TITLE
Backups stuck in "Activating" state should not be shown in the restore list

### ISSUE THREAD

**Issue by soumyalj** [rancher/rancher#18699]
Version : Master

Steps:
1. Create a 3 node cluster with invalid S3 credentials.
2. Perform few backups(B1, B2, B3), backups are stuck in "Activating" state
3. Edit and fix the S3 credentials. Wait for the cluster to become active.
4. Perform another backup (B4)
In the restore list, backups in "Activating" state are also listed in UI

The backups in "Activating" state should not be listed.

**Comment by westlywright** [comment]
I'm not sure why we'd want to hide activating items. This is not an error state and I'm unclear as to why we'd want to hide this from the user?

**Comment by vincent99** [comment]
They shouldn't be hidden from the list of backups, you just shouldn't be able to chose to restore one that isn't finished yet.

**Comment by davidnuzik** [comment]
Version: master-head (v2.3) (8/16/19)

I carefully followed the steps to reproduce in the issue description.
Upon fixing the s3 credentials and then doing one more backup and then viewed the "Restore Snapshot" in the UI -- now the ones that are in the "Activating" state are not accessible in the UI. I am only allowed to restore my Active/Ready one. I made sure my good snapshot made it in my s3 bucket.

![image](https://user-images.githubusercontent.com/45179589/63202959-42f0df00-c041-11e9-8bcb-88bfe00a5d96.png)

I restored this snapshot and ensured that this worked as expected without any issues.
